cmd/git.go

```diff
@@ -134,6 +132,44 @@ var diffSubCmd = &cobra.Command{
 	},
 }
 
+var renderPrTemplate = &cobra.Command{
+	Use:   "render-pr-template <repo> <template-name>",
+	Args:  cobra.MinimumNArgs(2),
+	Short: "Prints the given PR template for the repo  after rendering, verbatim as the PR body - template name to be given without the .tmpl extension",
+	Long: `Dumps a markdown formatted output of the PR body
+for the given pr template for the given repo - especially
+useful if used with policy-dump template as it will print
+the current release branches information for the given repo,
+<template-name> should be given without the .tmpl extension.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		repo := args[0]
+		tmpl := args[1]
+		owner, _ := cmd.Flags().GetString("owner")
+		r, err := git.Init(repo,
```

Comment:
Is the repo required to render a template? Is it not just stuff from the config file?